### PR TITLE
Make matchers in "should see" Cucumber step definitions case-insensitive

### PR DESCRIPTION
**Pull request by myabc**
Many Cucumber features were broken as a result of the styling change
introduced in fc47355. Cukes should test behaviour and semantics,
_not styling_. `text-transform: uppercase` does not convey semantic
meaning in the context of these particular headings.

Selenium will match text as it is rendered and visible to the
end-user. Thus, by default, we need to make the text matching in the
"should (not) see" step definitions less strict.

### CODE REVIEW

File: features/work_packages/moves/work_package_moves_new_copy.feature [hunk 1]
```diff
@@ -114,7 +114,6 @@ Feature: Copying a work package
 
      And I should see "project_2" within ".breadcrumb"
 
-  @javascript
```
Comment (myabc) [inline features/work_packages/moves/work_package_moves_new_copy.feature:117]: @tilt I'm not sure why this tag was here. This feature runs fine for me without JS.

File: features/step_definitions/web_steps.rb [hunk 2]
```diff
@@ -192,60 +192,38 @@ def with_scope(locator, options={})
...
 
 Then /^the "([^"]*)" field(?: within (.*))? should not contain "([^"]*)"$/ do |field, parent, value|
   with_scope(parent) do
     field = find_field(field)
-    if field.value.respond_to? :should_not
-      field.value.should_not =~ /#{value}/
-    else
-      assert_no_match(/#{value}/, field.value)
-    end
+    field.value.should_not =~ /#{value}/
```
Comment (tilt) [inline features/step_definitions/web_steps.rb:226]: are these respond_to checks obsolete by now?

File: features/work_packages/moves/work_package_moves_new_copy.feature (hunk 1 above)
Comment (tilt) [inline features/work_packages/moves/work_package_moves_new_copy.feature:117]: well

**Comment by tilt** [comment]
:+1: we should have done this at once

**Comment by myabc** [comment]
@tilt it took me some time to realise that fc473559 (and I think also a56b0e77b) caused the changes. too long…

File: features/step_definitions/web_steps.rb (hunk 2 above)
Comment (myabc) [inline features/step_definitions/web_steps.rb:226]: @tilt they're only needed if you're using Cucumber with Test Unit.

File: features/work_packages/moves/work_package_moves_new_copy.feature (hunk 1 above)
Comment (myabc) [inline features/work_packages/moves/work_package_moves_new_copy.feature:117]: well what?

Comment (tilt) [inline features/work_packages/moves/work_package_moves_new_copy.feature:117]: it was a blackout either by selenium or by me

File: features/step_definitions/web_steps.rb (hunk 2 above)
Comment (tilt) [inline features/step_definitions/web_steps.rb:226]: sounds like a horrible combination

**Comment by tilt** [comment]
Come on Travis

File: features/step_definitions/common_steps.rb [hunk 3]
```diff
@@ -77,4 +77,4 @@
 
 Then(/^"([^"]*)" should be the first row in table$/) do |name|
   should have_selector("table.list tbody tr td", :text => Regexp.new("#{name}"))
-end
+end
```
Comment (mfrister) [inline features/step_definitions/common_steps.rb:80]: There's a newline missing :)

Comment (tilt) [inline features/step_definitions/common_steps.rb:80]: you saved our lives ;)

**Pull request merged**